content/applications/sales/sales/send_quotations/create_quotations.rst

```diff
@@ -253,6 +253,24 @@ fields that can be configured.
   confirmation window appears, as well.
 - :guilabel:`Analytic Account`: Select an analytic account to apply to this customer/quotation.
 
+When :doc:`Debug mode is enabled <../../../general/developer_mode>`, two additional fields appear in
+this section:
+
+- :guilabel:`Invoicing Journal`: The sales order invoices in the journal selected. When no selection
+  is made, the sales journal with the lowest sequence is used.
+- :guilabel:`Invoice Status`: The following options are available in this field:
+
+  - :guilabel:`Nothing to Invoice`: default value if other conditions are not met, or if the sales
```

Comment:
it almost feels like full sntences after the colon- you can add a word to make them full sentences, like "The default value..." etc. Not necessary, just a suggestion!